vue/sbc-common-components/src/components/SbcHeader.vue

```diff
@@ -86,6 +139,14 @@ export default class SbcHeader extends Vue {
   goToUserProfile () {
     window.location.assign('/cooperatives/auth/userprofile')
   }
+
+  goToAccountSettings () {
```

Comment:
may be in another ticket , we have to implement the logic of checking if the user is in auth application , we can emit event or else we can assign the location ; so that the page wont be refreshed if user is in auth itself